homeassistant/components/control4/strings.json

```diff
@@ -0,0 +1,34 @@
+{
+  "config": {
+    "abort": {
+      "already_configured": "The Control4 controller associated with this account has already been configured."
+    },
+    "error": {
+      "cannot_connect": "Couldn't connect to the local controller. Check your hostname and login details.",
+      "invalid_auth": "Invalid Control4 username or password.",
+      "unknown": "An unknown error occured. Please report log info."
+    },
+    "step": {
+      "user": {
+        "data": {
+          "host": "Controller hostname",
```

Comment:
Interesting that we need to ask this. Can't we get the host from the cloud or via a scan?

Reply:
The Control4 controller does expose itself via SSDP, but it doesn't provide its own IP address in the SSDP `LOCATION` information. 

Is there a way to find the IP address a SSDP `NOTIFY` packet originates from without using the `LOCATION` info?

Reply:
Weird they don't have a location. I  think that we should be able to get the IP, but not sure if the current SSDP implementation offers that. It's fine like this for now, we'll ponder about it later :)